meta/implementation.md

```diff
@@ -73,4 +73,29 @@ you may define something like the following for the parameter:
 
 	]
 }
-```
\ No newline at end of file
+```
+
+## Date and Time manipulation
+
+Working with dates is a lot more complex than it seems to be at first sight. Issues arise especially with daylight saving times (DST), time zones, leap years and leap seconds.
+
+The date/time functions in openEO don't have any effect right now as only timestamps in UTC (with potential numerical time zone modifier) are supported.
+
+Leap years are implemented in a way that computations handle them gracefully. For example:
+
+- If you add a month to January, 31th, it will result in February 29th (leap year) or 28th (other years). This means for invalid dates we round down (or "snap") to the next valid date.
```

Comment:
"round down to the next ..." reads weird.  "previous" instead of "next" would only be slightly less weird. proposal:

> This means for invalid dates due to month overflow we round down (or "snap") to the last valid date of the month

Reply:
```suggestion
- If you add a month to January, 31th, it will result in February 29th (leap year) or 28th (other years). This means for invalid dates due to month overflow we round down (or "snap") to the last valid date of the month.
```